xhr/resources/send-after-setting-document-domain-window-helper.js

```diff
@@ -4,10 +4,13 @@ function assert_equals(value, expected) {
   }
 }
 
-function assert_throws(expected_exc, func) {
+function assert_throws_dom(expected_exc, func) {
   try {
     func.call(this);
   } catch(e) {
+    if (e.constructor.name != "DOMException") {
+      throw "Something that was not a DOMException was thrown";
```

Comment:
Might be nice to put unexpected error name in the error message:

    throw `Exception '${e.constructor.name || "unknown"}' that was not a DOMException was thrown`;

Reply:
Sure thing, will do.